MariaDB/server
# MDEV-28194: Command with path in maintainer script

- [x] *The Jira issue number for this PR is: MDEV-28194*

## Description
In Debian there shouldn't be postinst scripts with whole path included.
This fixes one that has on `mariadb-server-10.6.postinst`

## How can this PR be tested?
See Salsa-CI: TODO when done

Build manually:
```
debian/autobake-deb.sh
lintian ../*.chages
```
The shouldn't be line:
`W: mariadb-server-10.6: command-with-path-in-maintainer-script /usr/sbin/mariadbd (plain script) [postinst:118]`
Any more

## Basing the PR against the correct MariaDB version
- [x ] *This is a bug fix and the PR is based against the earliest branch in which the bug can be reproduced*

## Backward compatibility
Shouldn't make any difference
